demos/tabs/ajax.html

```diff
@@ -36,7 +53,6 @@
 
 <div class="demo-description">
 <p>Fetch external content via Ajax for the tabs by setting an href value in the tab links.</p>
-<p>Tabs 3 and 4 demonstrate slow-loading and broken AJAX tabs, and how to handle serverside errors in those cases. Note: These two require a webserver to interpret PHP. They won't work from the filesystem.</p>
```

Comment:
I've restored the first sentence and updated the wording to better reflect the current situation. I had initially removed this since the tabs already have descriptive names "Tab 3 (slow)" and "Tab 4 (error)".